src/Core/Geographic/Coordinates.js

```diff
@@ -136,7 +136,8 @@ function _convert(coordsIn, newCrs, target) {
         }
     } else {
         if (coordsIn.crs === 'EPSG:4326' && newCrs === 'EPSG:4978') {
-            const cartesian = ellipsoid.cartographicToCartesian(coordsIn);
+            target._normal = ellipsoid.geodeticSurfaceNormalCartographic(coordsIn);
```

Comment:
Doing:
```js
target._normal = coordsIn.geodesicNormal;
```
Would avoid recomputing the normal if it was already done for `coordsIn`

Reply:
done